modules/exploits/windows/persistence/assistive_technology.rb

```diff
@@ -0,0 +1,115 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::EXE
+  include Msf::Exploit::Local::Persistence
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Post::Windows::Registry
+  include Msf::Post::Windows::Priv
+
+  AT_REG_PATH = 'HKLM\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Accessibility\\ATs'
+  STARUP_REG_PATH = 'HKCU\\Software\\Microsoft\\Windows NT\\CurrentVersion\\Accessibility'
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Assistive Technologies Persistence',
+        'Description' => %q{
+          This module achieves persistence by registering a custom Assistive Technology (AT) in the Windows registry.
+          Then it configures the system to launch the AT executable during user logon or desktop switch (such as with
+          an admin prived program).
+          Requires Windows 8 or higher and administrative privileges.
+        },
+        'Author' => ['h00die'],
+        'Platform' => ['win'],
```

Comment:
```suggestion
        'Platform' => ['win'],
        'Arch' => [ARCH_X64, ARCH_X86, ARCH_AARCH64],

```
If you don't specify an arch, it appears to default to x86-only.  Or did you mean to limit it to only x86 payloads?